Assets/GlitchEscape/Scripts/Player/Controls/PlayerJumpController_Alt.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
```

Comment:
don't make this a separate file - make your changes just to PlayerJumpController.cs (so we can see a diff)